### PR TITLE
Update android-platform-tools to 26.0.2

### PR DESCRIPTION
**Pull request by linbo0518**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

File: Casks/android-platform-tools.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@
   sha256 'a6d0504e560713af2a3ae71449bcadf011b50ba78f7bf303a9d6d69bf855c73f'
 
   # google.com/android/repository/platform-tools was verified as official when first introduced to the cask
-  url 'https://dl.google.com/android/repository/platform-tools_r26.0.2-darwin.zip'
```
Comment (commitay) [inline Casks/android-platform-tools.rb]: Is there a website with the versioned download URL?

Comment (linbo0518) [reply Casks/android-platform-tools.rb]: yes, #{version} maybe better

Comment (linbo0518) [reply Casks/android-platform-tools.rb]: This is more convenient to use cask-repair to upgrade version

Comment (commitay) [reply Casks/android-platform-tools.rb]: Where did you get the versioned URL from?

Comment (linbo0518) [reply Casks/android-platform-tools.rb]: the latest version of platform-tool have some bugs on my computer, so I downloaded earlier version and found the regularity of this URL

**Review by commitay — APPROVED** [review]

**Pull request merged**